site/_posts/2017-01-30-keeping-up-with-the-conferences.md

```diff
@@ -0,0 +1,42 @@
+
+Ansible is great and we would like to have better integration with our automation. [James](https://github.com/ManageIQ/manageiq/pull/13630) is one of the people working on this very hard. By [migrating](https://github.com/ManageIQ/manageiq/pull/13630) the `ConfigurationManager` of Ansible Tower to the new `AutomationManager` we are one step closer in achieving this goal.
+
+## Fixed
+
+As we are trying to move the VMware provider to its new repo, some paths in requiring helpers weren't been adjusted properly. Thanks to [Adam](https://github.com/agrare) this is now [fixed and working](https://github.com/ManageIQ/manageiq-providers-vmware/pull/2). These things are expected consequences of the moving, but they can be found and fixed quickly.
+
+We are proudly using [Sauce Labs](https://saucelabs.com/) for continuous integration of our Service UI. This feature can be enabled for a fork of the repository by turning on Travis tests. However, when you forgot your Sauce Labs credentials, the tests were running and failing. Thanks to [Chris Hale](https://github.com/chalettu) this is now [fixed](https://github.com/ManageIQ/manageiq-ui-service/pull/460).
+
+## New
+
+Collecting metrics is useful, but storing them in an SQL database is not always *The Right Way*&trade;. [Jason](https://github.com/Fryguy) intrudoced a new model called [ActiveMetrics](https://github.com/ManageIQ/manageiq/pull/13436) that is similar to `ActiveRecord` but it has adapters to databases tuned for collecting metrics in a high frequency.
+
+We introduced the new dashboard type summary screens a few months ago. These are giving you a better overview of a given entity by showing **more charts** and less tables. Thanks to [Zita](https://github.com/ZitaNemeckova) now you have a [configuration option in the UI](https://github.com/ManageIQ/manageiq-ui-classic/pull/141) to set the dashboards as the default view across the UI.
+
+## Deleted
+
+[Brandon](https://github.com/bdunne) found out that some constants were not used anywhere. These were connected with Automation at some point, but no longer used, so it was safe to [get rid of them](https://github.com/ManageIQ/manageiq/pull/13665).
+
+[Milan](https://github.com/mzazrivec) is making our internationalization support better and better, which means that we can delete more legacy code used to maintain compatibility. A good example of this is the use of `PostponedTranslation` which is no longer required in `UiConstants` so [it has been deleted](https://github.com/ManageIQ/manageiq-ui-classic/pull/241).
+
+## Wrapping up
+This was a quick overview of what have we done last week, but feel free to check out the complete list for each repository. If you want to meet us in person, we will be attending on [FOSDEM](https://fosdem.org/2017/) this weekend in Brussels, Belgium. Search for the people in ManageIQ T-shirts!
```

Comment:
>we will be attending on [FOSDEM](https://fosdem.org/2017/) this weekend in Brussels, Belgium

I think the 'on' is not needed here.

we will be attending [FOSDEM](https://fosdem.org/2017/) this weekend in Brussels, Belgium